src/Locales/zh_CN.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+// locale: chinese (zh-cn)
+// author: suupic : https://github.com/suupic
+// author: Zeno Zeng : https://github.com/zenozeng
+// author: Senorsen : https://github.com/Senorsen
+
+return array(
+    "months"        => explode('_', '一月_二月_三月_四月_五月_六月_七月_八月_九月_十月_十一月_十二月'),
+    "monthsShort"   => explode('_', '1月_2月_3月_4月_5月_6月_7月_8月_9月_10月_11月_12月'),
+    "weekdays"      => explode('_', '星期日_星期一_星期二_星期三_星期四_星期五_星期六'),
+    "weekdaysShort" => explode('_', '周日_周一_周二_周三_周四_周五_周六'),
+    "weekdaysMin"   => explode('_', '日_一_二_三_四_五_六'),
+    "calendar"      => array(
+        "sameDay"  => function() {
```

Comment:
function gets passed a ```Moment``` object. So you should not use ```$this``` but actually do the following:

```php
// e.g. for "sameDay"
"sameDay"  => function(Moment\Moment $moment) {
    return $moment->getMinute() === 0 ? '[今天]Ah[点整]' : '[今天]LT';
}
```